pkg/bootstrap/docker/dockermachine/helper.go

```diff
@@ -147,10 +150,32 @@ func Client(name string) (*docker.Client, error) {
 		client, err = docker.NewVersionedClient(dockerHost, "")
 	}
 	if err != nil {
-		return nil, errors.NewError("could not get Docker client for machine %s", name).WithCause(err)
+		return nil, nil, errors.NewError("could not get Docker client for machine %s", name).WithCause(err)
 	}
 	client.SkipServerVersionCheck = true
-	return client, nil
+
+	var httpClient *http.Client
+	if len(certPath) > 0 {
+		tlscOptions := tlsconfig.Options{
+			CAFile:             filepath.Join(certPath, "ca.pem"),
+			CertFile:           filepath.Join(certPath, "cert.pem"),
+			KeyFile:            filepath.Join(certPath, "key.pem"),
+			InsecureSkipVerify: tlsVerify,
```

Comment:
@kargakis yes thank you for catching that
@mfojtik do you want another pull or can you make the change on your branch?